oprf/oprf_test.go

```diff
@@ -0,0 +1,230 @@
+package oprf
+
+import (
+	"bytes"
+	"encoding/hex"
+	"testing"
+
+	"github.com/cloudflare/circl/oprf/group"
+)
+
+func TestServerSetUp(t *testing.T) {
+	srv, err := NewServer(OPRFP256)
+	if err != nil {
+		t.Fatal("invalid setup of server: " + err.Error())
+	}
+	if srv == nil {
+		t.Fatal("invalid setup of server: no server.")
+	}
+
+	if srv.Keys == nil {
+		t.Fatal("invalid setup of server: no keypair")
+	}
+}
+
+func TestClientSetUp(t *testing.T) {
+	client, err := NewClient(OPRFP256)
+	if err != nil {
+		t.Fatal("invalid setup of client: " + err.Error())
+	}
+	if client == nil {
+		t.Fatal("invalid setup of client: no server.")
+	}
+}
+
+func TestClientBlind(t *testing.T) {
+	client, err := NewClient(OPRFP256)
+	if err != nil {
+		t.Fatal("invalid setup of client: " + err.Error())
+	}
+
+	token, bToken, err := client.Blind([]byte{00})
```

Comment:
(just an idea) 
an interesting test would be to show the ability that client can do `unblind(blind(x)) == x` .
but it's not possible with the current api.

Reply:
Let me see how can I add it ;)

Reply:
I included VerifyFinalize now, which does a nicer check..